JS Lab/Problem2.js

```diff
@@ -0,0 +1,31 @@
+(function(){
+    function Crate(weight, color){
+       this.weight = weight;
+       this.color = color;
+    }
+    var n = prompt("Enter an integer:");
+    var array = [n];
+    for(var i = 0; i < n; i++){
+        array[i] = [n];
```

Comment:
This is not the correct way to generate two dimensional array of nxn. All you are creating is n*1 sized array.